sympy/functions/elementary/exponential.py

```diff
@@ -597,6 +594,8 @@ def _eval_expand_log(self, deep=True, **hints):
             p = perfect_power(int(arg))
             if p is not False:
                 return p[1]*self.func(p[0])
+        elif arg.is_Rational:
+            return log(arg.p) - log(arg.q)
```

Comment:
If you are wondering if this is right or not, I would think so -- if someone wants to expand logs, ratios (where possible) would be rewritten as you have done.